### ISSUE TITLE
Page Template or Sidebar Issue

### ISSUE THREAD

**Issue by WMDesign** [ThemeFuse/Unyson#3299]
I'm using page templates for three different types of pages on my site.  Each of these page types will be used between 10 - 150 times.  Right now I'm using them about three times each, getting everything set up.  

I just encountered an issue where I needed to change a widget in each of the page types - and I ended up having to do it for each of the pages IN those types as well because I wasn't sure where to go to edit the 'template'?  I'm not even sure if that's possible.

Basically, each of these page types has a menu on the left and a content area on the right.  I'm using a 1/4 3/4 layout, with a widget area in the 1/4 and a text area in the 3/4.  I had to go to the widget area on all 12 pages and change the menu it was pointing to.  I'm wondering if there would be an easier way to do this, possibly with sidebars?

Manually editing 12 pages was a bit of a pain, but manually editing 300 pages would be a NIGHTMARE.

Thanks for any help =)

**Comment by ViorelEremia** [comment]
There is no way to do that. Use sidebars to manipulate all pages. but if you are a developer you can try and analyze code below; be careful to use this code it is too dangerous you can break all pages.
```
add_action( 'after_setup_theme', function() {
	$pages = get_pages();

	foreach ( $pages as $page ) {

		$data = fw_get_db_post_option( $page->ID, 'page-builder' );
		$els = isset( $data['json'] ) ? json_decode( $data['json'], true ) : array();

		if ( empty( $data['builder_active'] ) || empty( $els ) ) {
			continue;
		}

		// $els contains all shortcodes perfom here actions on them
		$data['json'] = json_encode( $els );
		fw_set_db_post_option( $page->ID, $data );
	}
}, 11 );
```
